src/main/java/entity/CommonUser.java

```diff
@@ -5,19 +5,27 @@
  */
 public class CommonUser implements User {
 
-    private final String name;
+    private final String username;
     private final String password;
     private final int accessLevel;
+    private final String userID;
+    private final String birthDate;
+    private final String fullName;
+    private final String email;
 
-    public CommonUser(String name, String password) {
-        this.name = name;
+    public CommonUser(String name, String password, String userID, String birthDate, String fullName, String email) {
+        this.username = name;
         this.password = password;
+        this.userID = userID;
+        this.birthDate = birthDate;
+        this.fullName = fullName;
+        this.email = email;
         this.accessLevel = 0;
```

Comment:
Pass `0` through the constructor and assign it to this.accessLevel (i.e. `this.accessLevel = accessLevel`). We'll create users through factories, so the initializations should go there.